### PR TITLE
UI cleanup

### PR DESCRIPTION
**Pull request by RainingComputers**

### CODE REVIEW

**Comment by RainingComputers** [comment]
Some checked button bugs still remain, but easy to fix

File: meiki_ui/src/components/app/LogoutModal.svelte [hunk 1]
```diff
@@ -1,17 +1,19 @@
 <script lang="ts">
-    import { goto } from "$app/navigation"
+    import { createEventDispatcher } from "svelte"
     import { logout } from "$lib/api/user"
     import Button, { ButtonType } from "$cmp/Button.svelte"
     import Logo from "./Logo.svelte"
 
+    const dispathEvent = createEventDispatcher()
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/app/LogoutModal.svelte:7]: typo

File: meiki_ui/src/routes/notes/index.svelte [hunk 2]
```diff
@@ -80,19 +81,19 @@
         currentNoteText = undefined
     }
 
-    function onTextChange(event: any) {
+    function onTextChange(event: CustomEvent<{ text: string }>) {
```
Comment (alexsanjoseph) [inline meiki_ui/src/routes/notes/index.svelte:83]: CustomEvent is a thing huh

**Comment by alexsanjoseph** [comment]
LGTM except small issues

**Pull request merged**